packages/scandipwa/src/store/MyAccount/MyAccount.dispatcher.js

```diff
@@ -203,6 +203,52 @@ export class MyAccountDispatcher {
         );
     }
 
+    /**
+     * Resend confirmation email
+     * @param {{email: String}} [options={}]
+     * @memberof MyAccountDispatcher
+     */
+    resendConfirmation(options = {}, dispatch) {
+        const mutation = MyAccountQuery.getResendConfirmationMutation(options);
+
+        return fetchMutation(mutation).then(
+            /** @namespace Store/MyAccount/Dispatcher/MyAccountDispatcher/resendConfirmation/fetchMutation/then */
+            (data) => {
+                const { resendConfirmationEmail: { status = '' } } = data;
+
+                switch (status) {
+                case 'account_confirmation_not_required':
```

Comment:
Please put constant string values to shareable constants.
This will allow us to reuse them later and makes it impossible to make a typo.

Just to be clear. Machine names like this should be considered to be moved to the constants.
Text messages, titles, etc. mean to be read by humans may remain as is.
And please don'ttouch the `showNotification` function first argument. We already handled it in the TS migration branch. If you move this to the constant, it will lead to the merge conflicts.